V0_9UpgradeMessageExe/Program.cs

```diff
@@ -22,7 +22,7 @@ namespace SonarQube.V0_9UpgradeMessageExe
      * message and exits with an error code that will cause the build to fail.
      */
 
-    public class Program
+    public static class Program
     {
         static int Main(string[] args)
         {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Parameter 'args' of 'Program.Main(string[])' is never used. Remove the parameter or use it in the method body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=fxcop%3AReviewUnusedParameters)